src/_cffi_src/openssl/dh.py

```diff
@@ -227,4 +229,17 @@
     return DH_check(dh, ret);
 }
 #endif
+
+/* These functions were added in OpenSSL 1.1.0f commit d0c50e80a8 */
+#if CRYPTOGRAPHY_OPENSSL_LESS_THAN_110PRE6
```

Comment:
I am currently failing on building at versions which are older then 1.1.0-pre6. When 1.1.0f will be released it will contain d0c50e80a8  thus this condition is valid.
I will try configuring `Cryptography_d2i_DHxparams_bio` and see if that helps...